lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,28 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # class to feel yourself real driver
+  class Car
+    attr_accessor :color, :model, :current_speed, :sets, :year
```

Comment:
i catch an error in initialize that i don't have a method color= and other, only sets doesn't need a accessor